### PR TITLE
Add blog article about feeds

### PR DESCRIPTION
**Pull request by TheAssassin**
CC @TobiGr

### CODE REVIEW

File: _data/images.yml [hunk 1]
```diff
@@ -36,3 +36,10 @@ speech-bubbles:
   author: María Florencia
   origin: https://commons.wikimedia.org/wiki/File:Noun_project_2472.svg
   license: CC0 1.0
+
+feed-icon:
+  url: feed-icon.svg
+  type: jpg
+  author: Clker Free Vector Images
```
Comment (TobiGr) [inline _data/images.yml]: please add the file size. It now says that the svg has a size of 5 bytes :D

**Comment by TobiGr** [comment]
```
<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0">
<channel>
<title>NewPipe Blog</title>
<description>The latest blog posts</description>
<link>/</link> 
...
```
That's what the feed looks which is available on https://newpipe.schabi.org/blog/feeds/news.rss. For some reason the links get not displayed there, but when I build it locally, they do. I'll investigate this. Besides this we can merge it.

File: _data/images.yml (hunk 1 above)
Comment (TobiGr) [reply _data/images.yml]: I'll update the readme soon and include a hint about this.

**Comment by TheAssassin** [comment]
@TobiGr I already mentioned it's because the web server returns the wrong MIME type. On your system, you use that node.js based thingy jekyll comes with. Our real deployment uses NGINX and serves the pre-built static HTML pages only. With some minor configuration fixes in the NGINX configs, we can "fix" that issue as

**Pull request merged**